WireUI/Sources/WireSidebar/Views/SidebarView.swift

```diff
@@ -90,58 +97,50 @@ public struct SidebarView<AccountImageView>: View where AccountImageView: View {
     }
 
     @ViewBuilder
-    private var profileSwitcher: some View {
+    private var accountInfoView: some View {
         if let accountInfo {
-            SidebarProfileSwitcherView(
+            SidebarAccountInfoView(
                 displayName: accountInfo.displayName,
                 username: accountInfo.username,
                 accountImageView: { accountImageView(accountInfo.accountImage, accountInfo.availability) }
             )
         }
     }
 
-    @ViewBuilder @MainActor
-    private var menuItems: some View {
+    @ViewBuilder
+    private var scrollableMenuItems: some View {
         VStack(alignment: .leading, spacing: 0) {
-            Text("sidebar.conversation_filter.title", bundle: .module)
-                .wireTextStyle(.h2)
-                .padding(.horizontal, 8)
-                .padding(.vertical, 12)
-            ForEach([SidebarConversationFilter?.none] + SidebarConversationFilter.allCases, id: \.self) { conversationFilter in
-                conversationFilter.label(iconSize, isActive: self.conversationFilter == conversationFilter) {
-                    self.conversationFilter = conversationFilter
-                }
+            menuItemHeader("sidebar.conversation_filter.title", addTopPadding: false)
+            let conversationFilters = [SidebarMenuItem.all, .favorites, .groups, .oneOnOne, .archive]
+            ForEach(conversationFilters, id: \.self) { conversationFilter in
+                menuItem(conversationFilter)
             }
 
-            Text("sidebar.contacts.title", bundle: .module)
-                .wireTextStyle(.h2)
-                .padding(.horizontal, 8)
-                .padding(.vertical, 12)
-                .padding(.top, 12)
-            SidebarMenuItem(
-                icon: "person.badge.plus",
-                iconSize: iconSize
-            ) {
-                Text("sidebar.contacts.connect.title", bundle: .module)
-            } action: {
-                connectAction()
-            }
+            menuItemHeader("sidebar.contacts.title")
```

Comment:
question: we don't use SwiftGen localizable constants here?

Reply:
I couldn't get it to work with Swift packages. Then I came to an agreement with my PR reviewer that I use String catalogs, which give visual feedback if something is wrong (like unused keys in the catalog).
Moreover I've added the launch argument
`-NSShowNonLocalizedStrings YES`
which not only displays a warning to the console but also shows all unlocalized strings in capital letters.
<img width="1115" alt="Screenshot 2024-09-24 at 09 24 08" src="https://github.com/user-attachments/assets/c72f5e32-332c-41dc-8da9-51c7e933df3f">